package.txt

```diff
@@ -1,6 +1,6 @@
 NAME=nano
-VERSION=2.9.3
-REVISION=2
-SRC=https://www.nano-editor.org/dist/v2.9/nano-2.9.3.tar.xz
-SRCMD5=6dd37f010d6a985bf0e5f8ed091ff919
-CLEANUP=yes
\ No newline at end of file
+VERSION=3.0
+REVISION=0
```

Comment:
The REVISION field is a monotonic integer increased by one for each update to the package. This allows shedmake to see that an update is available even when the packaged software itself is not being updated to a later version, accommodating patches and packaging updates (e.g. nano-2.9.3-2 and nano-2.9.3-3).
In this case, the REVISION should be bumped to 3.